Documentation meta-descriptions review N° 4 (#5558)
wazuh/wazuh-documentation: source/user-manual/reference/tools/cluster-control.rst

.. Copyright (C) 2015, Wazuh, Inc.

.. meta::

>   :description: The cluster_control program allows you to manage the cluster from any manager. Learn more about it in this section of the Wazuh documentation.

Reviewer: ```suggestion
  :description: Manage the Wazuh cluster from any manager using the cluster_control program. Learn more about it in this section of the Wazuh documentation.
```
